### PR TITLE
Create ci.yml to run tests

### PR DESCRIPTION
**Pull request by Vahila**
Created a ci.yml file to run Maven tests. The yml file is configured to run on 'push'.

### CODE REVIEW

**Review by nbhoski — COMMENTED** [review]
Do you have Azure licence ? may be it would let us add mac os

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  test:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Run Maven tests
```
Comment (nbhoski) [inline .github/workflows/ci.yml:16]: We could use verify which also runs static code analyzer and spotbugs etc

Comment (Vahila) [reply .github/workflows/ci.yml:16]: Done.

**Comment by Vahila** [comment]
> Do you have Azure licence ? may be it would let us add mac os

macos runners are available in GitHub actions as well. I added it as a runner as well.

**Review by nbhoski — APPROVED** [review]
Ship it !

**Pull request merged**